packages/components/src/CircularProgress/CircularProgress.test.js

```diff
@@ -1,31 +1,35 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { screen, render, configure } from '@testing-library/react';
 
 import CircularProgress from './CircularProgress.component';
 
+configure({ testIdAttribute: 'data-test' });
+
 describe('CircularProgress', () => {
 	it('should render by default at default size', () => {
-		const wrapper = renderer.create(<CircularProgress />).toJSON();
-		expect(wrapper).toMatchSnapshot();
+		render(<CircularProgress />);
+		expect(screen.getByTestId('circular-progress')).toBeInTheDocument();
```

Comment:
toBeInTheDocument checks only for the availability of the element in DOM
so, i'd prefer "toBeVisible()" here and in other places